.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2.1
+orbs:
+  slack: circleci/slack@3.4.2
+
+jobs:
+  build:
+    docker:
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:stitch-tap-tester
+    steps:
+      - checkout
+      - run:
+          name: 'Setup virtual env'
+          command: |
+            python3 -mvenv /usr/local/share/virtualenvs/tap-crossbeam
+            source /usr/local/share/virtualenvs/tap-crossbeam/bin/activate
+            pip install -U pip setuptools
+            pip install .[test]
+      - run:
+          name: 'JSON Validator'
+          command: |
+            source /usr/local/share/virtualenvs/tap-tester/bin/activate
+            stitch-validate-json tap_crossbeam/schemas/*.json
+      - run:
+          name: 'pylint'
+          command: |
+            source /usr/local/share/virtualenvs/tap-crossbeam/bin/activate
+            pylint tap_crossbeam --disable 'broad-except,chained-comparison,empty-docstring,fixme,invalid-name,line-too-long,missing-class-docstring,missing-function-docstring,missing-module-docstring,no-else-raise,no-else-return,too-few-public-methods,too-many-arguments,too-many-branches,too-many-lines,too-many-locals,ungrouped-imports,wrong-spelling-in-comment,wrong-spelling-in-docstring,bad-whitespace'
+      - slack/status:
+          channel: 'stitch-tap-tester-tests'
+          mentions: "${CIRCLE_USERNAME}"
+          fail_only: true
+          only_for_branches: master
+          failure_message: |
+            :red_circle: ${CIRCLE_JOB} failed!
```

Comment:
This seems strange to break from our usual `slack/notify-on-failure`

Reply:
This is just cut and paste from how-to-sources, which might be out of date